compiler/qsc_qasm3/src/tests/statement/gate_call.rs

```diff
@@ -6,6 +6,64 @@ use expect_test::expect;
 use miette::Report;
 use qsc::target::Profile;
 
+#[test]
+fn u_gate_can_be_called() -> miette::Result<(), Vec<Report>> {
+    let source = r#"
+        qubit q;
+        U(1.0, 2.0, 3.0) q;
+    "#;
+
+    let qsharp = compile_qasm_to_qsharp(source)?;
+    expect![
+        r#"
+        operation U(theta : Double, phi : Double, lambda : Double, qubit : Qubit) : Unit is Adj + Ctl {
+            body ... {
+                Rz(lambda, qubit);
+                Ry(theta, qubit);
+                Rz(phi, qubit);
+                R(PauliI, -lambda - phi - theta, qubit);
+            }
+            adjoint auto;
+            controlled auto;
+            controlled adjoint auto;
+        }
+        let q = QIR.Runtime.__quantum__rt__qubit_allocate();
+        U(1., 2., 3., q);
+        "#
+    ]
+    .assert_eq(&qsharp);
+    Ok(())
+}
+
+#[test]
+fn gphase_gate_can_be_called() -> miette::Result<(), Vec<Report>> {
```

Comment:
See the comment in the lowerer. This test got copied from above and the expect is wrong and we shouldn't be using a qubit.